frontend/src/Components/LoginPage/LoginPage.module.css

```diff
@@ -89,3 +83,41 @@
   grid-column: 1 / 3;
   justify-self: center;
 }
+
+.sessionContainer {
+  display: grid;
+  gap: 1ch;
+  grid-template-columns: 1fr min-content;
+  grid-column: 2 / 3;
+  align-self: center;
+  align-items: center;
+}
+
+.sessionLink {
+  color: var(--primary);
+  min-width: 19ch;
+}
+
+.refreshButton {
+  color: var(--primary);
+  fill: var(--primary);
+  border: unset;
+  border: 1px solid var(--primary);
+  background: var(--secondary);
+  border-radius: var(--border-radius);
+  cursor: pointer;
+  height: 22px;
+  width: 22px;
+  padding: 2px;
+  display: block;
+  line-height: unset;
```

Comment:
That feels dangerous to me, but maybe I am wrong. The parent does not really have an explicit line-height, so it is set to "normal", which is again something browser dependent. Considering that having a line-height that is too much would look bad, why not set it explicitly e.g. to 1, which appears to look about right?

Reply:
Going with the height of the icon, i.e., 16px.